insta/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { CardText } from 'reactstrap';
+import PropTypes from 'prop-types';
+
+const Comment = props => {
+  return (
+    <div className="comment">
+      <CardText><span className="comment-username">{props.comment.username} </span>{props.comment.text}      <span className="posted">{props.comment.posted}</span></CardText>
+
+    </div>
+  );
+};
+
+Comment.propTypes = {
```

Comment:
Yes!!!!!! You're already using props-types! Don't stop! It can be tedious but it's good to have that way you know what data you're playing with.